Dockerfile.cli.jdk8

```diff
@@ -0,0 +1,54 @@
+# This image will be published as dspace/dspace-cli
+# See https://dspace-labs.github.io/DSpace-Docker-Images/ for usage details
+#
+# This version is JDK8 compatible
+# - openjdk:8-jdk
+# - ANT 1.10.5
+# - maven:3-jdk-8
+# - note:
+# - default tag for branch: dspace/dspace-cli: dspace/dspace-cli:dspace-7_x
+
+# Step 1 - Run Maven Build
+FROM dspace/dspace-dependencies:dspace-7_x as build
+ARG TARGET_DIR=dspace-installer
+WORKDIR /app
+
+# The dspace-install directory will be written to /install
+RUN mkdir /install \
+    && chown -Rv dspace: /install \
+    && chown -Rv dspace: /app
+
+USER dspace
+
+# Copy the DSpace source code into the workdir (excluding .dockerignore contents)
+ADD --chown=dspace . /app/
+COPY dspace/src/main/docker/local.cfg /app/local.cfg
+
+# Build DSpace.  Copy the dspace-install directory to /install.  Clean up the build to keep the docker image small
+RUN mvn package && \
```

Comment:
as with the 6.x PR, though we have different profiles now with mvn in 7+, can we exclude the webapp to keep compile time faster since we shouldn't need it for the special CLI image?